test/studies/isx/.gitignore

```diff
@@ -0,0 +1 @@
+*.good
```

Comment:
For something that we want to ignore everywhere, like a.out, it completely makes sense to only mention it in the top-level file.  For something that we can't ignore in most places (like *.good,) putting a long relative path into a top-level .gitignore seems harder to maintain than a per-directory .gitignore (which better supports moving the test around, removing the entry if the test is removed in the future, etc.)